tools-local/tasks/tasks.proj

```diff
@@ -5,6 +5,8 @@
                       Condition="'$(TargetOS)' != 'Android'" />
     <ProjectReference Remove="$(MSBuildThisFileDirectory)mobile.tasks\AppleAppBuilder\AppleAppBuilder.csproj"
                       Condition="'$(TargetOS)' != 'iOS' and '$(TargetOS)' != 'tvOS'" />
+    <ProjectReference Remove="$(MSBuildThisFileDirectory)mobile.tasks\WasmAppBuilder\WasmAppBuilder.csproj"
+                      Condition="'$(TargetOS)' != 'Browser'" />
```

Comment:
question, shouldn't this be `$(TargetOS)' == 'Browser'` or does this need to be always except when targeting Browser?